src/s2d_engine/s2d_parse.c

```diff
@@ -273,14 +273,15 @@ void s2d_type_print_alloc(int x, int y, int align, const char *str, int *pos) {
 	}
 }
 
-// broken atm
 static int s2d_width(const char *str, int line, int len) {
 	char *p = str;
 	int tmp_len = 0;
 	int curLine = 0;
 	int width = 0;
 	int scale = 1;
 
+	u8 *ktable = seg2virt(s2d_kerning_table);
```

Comment:
what is this function/macro thing its weird

Reply:
homebrew compatibility where you can plug your own function into config.h

Reply:
coming back to this wouldnt it be better to have the function name still be segmented_to_virtual (same for all the config macros to have the default name the same as the sm64 name) so its less confusing and is still configurable for homebrew